### ISSUE TITLE
crash on startup in 1.15.2

### ISSUE THREAD

**Issue by LlubNek** [LexManos/CobbleForDays#6]
Got a "Registry Object not present" error when starting Minecraft.

CobbleForDays-1.1.1
forge-31.1.0
minecraft-1.15.2

```
java.lang.NullPointerException: Registry Object not present
	at java.util.Objects.requireNonNull(Objects.java:228) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.RegistryObject.get(RegistryObject.java:91) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.RegistryObject.isPresent(RegistryObject.java:115) ~[?:?] {re:classloading}
	at net.minecraftforge.lex.cfd.CobbleForDays$$Lambda$5106/1131724881.test(Unknown Source) ~[?:?] {}
...
```
[latest.log](https://github.com/LexManos/CobbleForDays/files/4164436/latest.log)
[crash-2020-02-06_05.00.06-client.txt](https://github.com/LexManos/CobbleForDays/files/4164438/crash-2020-02-06_05.00.06-client.txt)

**Comment by LexManos** [comment]
You have QUITE a few mods installed that are throwing errors all over the place. And I can't reproduce this with just CFD. So... Let me know if you can repro on sane env.

**Comment by Tander** [comment]
Had the same issue. These mods were updated before it broke:

PowderPower-1.15.2-3.1.57.jar
PickleTweaks-1.15.2-4.0.2.jar
performant-1.15-2.06m.jar
morevanillalib-1.15.2-1.2.2.jar
MekanismTools-1.15.2-9.9.20.412.jar
MekanismGenerators-1.15.2-9.9.20.412.jar
Mekanism-1.15.2-9.9.20.412.jar
jei-1.15.2-6.0.0.3.jar
EnchantmentDescriptions-1.15.2-2.0.8.jar
CyclopsCore-1.15.2-1.6.2.jar
Cucumber-1.15.2-3.0.5.jar
Controlling-6.1.4.jar

I tried to put them in client one by one and got fully working client with exactly the same mod. Magic.

[crash-2020-04-16_03.29.22-client.txt](https://github.com/LexManos/CobbleForDays/files/4484322/crash-2020-04-16_03.29.22-client.txt)
